packages/project/src/__tests__/validation.test.ts

```diff
@@ -183,186 +220,240 @@ describe('validation', () => {
-                    params: { limit: 1 },
-                    schemaPath: '#/properties/fields/minItems',
-                  },
-                ],
-              },
-              schemaPath: '#/properties/fields/errorMessage',
-            },
-          ],
-          message: 'data/fields must specify 1-25 fields',
-          formattedData: {
-            fields: [],
-            version: '1.0.0',
-          },
+              const data = {};
+              expect(
+                validateProjectData(data as ProjectOrAny<typeof data>)
+              ).toStrictEqual<ReturnType<typeof validateProjectData>>(
+                buildExpectedError({
+                  errorCode: 'projectUnknownError',
+                  dataPath: '',
+                })
+              );
+            });
         });
       });
-    });
-    describe('validation for the `historicLandManagement` property', () => {
-      describe('when the type is excluded or null', () => {
-        it('should return true for validation', () => {
-          const data: Project = {
-            version: '0.1.0',
-            fields: [
-              {
-                acres: 174.01,
-                historicLandManagement: null,
-                regenerativeStartYear: 2015,
-                fieldName: 'zyt0f1mnasi',
-                geojson: {
-                  coordinates: [
-                    [
-                      [
-                        [-102.02569636144796, 41.16245691933347],
-                        [-102.02423723974385, 41.1631353976904],
-                        [-102.02616843023458, 41.16184305191021],
-                        [-102.02569636144796, 41.16245691933347],
-                      ],
-                    ],
-                  ],
-                  type: 'MultiPolygon',
-                },
-                cropYears: [],
-              },
-            ],
-          };
-          expect(validateProjectData(data)).toStrictEqual({
-            valid: true,
-            errors: null,
-            message: 'No errors',
-            formattedData: data,
+      describe('`fields`', () => {
+        describe('projectFieldsMinimumItemsError', () => {
+          describe('when the number of fields is less than 1', () => {
+            it('should throw a validation error', () => {
+              const data = {
+                ...BASIC_UNFORMATTED_VALID_PROJECT,
+                fields: [] as Project['fields'],
+              };
+              expect(validateProjectData(data)).toStrictEqual<
+                ReturnType<typeof validateProjectData>
+              >(
+                buildExpectedError({
+                  errorCode: 'projectFieldsMinimumItemsError',
+                  dataPath: '/fields',
+                })
+              );
+            });
+          });
+        });
+        describe('projectFieldsMaximumItemsError', () => {
+          describe('when the number of fields is more than 25', () => {
```

Comment:
Is 25 fields also a limit in the soil metric model? I though we'd imposed that limit on ourselves to keep the sheet workable...

Reply:
no, this should definitely be removed if we don't intend to ever import to the sheet